### DISCUSSION TITLE
Funktionen beim Verlassen eines Geofence

### DISCUSSION THREAD

**Discussion by albi** [bassmaster187/TeslaLogger#968] [Ideas]
Nachdem ich zu Hause das Ladelimit immer auf einen bestimmten Wert einstellen lasse via TeslaLogger, wäre es aber auch genauso gut beim Verlassen dieses Geofence einen anderen Wert einstellen zu lassen. Wäre das möglich?

**Comment by superfloh247** [comment]
Hmm, also wenn da nicht das Funkloch beim Umschalten zwischen WLAN und Mobilfunk dazwischenfunkt (haha), dann sollte das gehen.

Auf das Ereignis "P->D bzw P->R" reagieren, Geofence checken, nach "charge limit on leave" suchen, setzen.

Könnte so aussehen: +clol:85 in der Config.
